content/english/Kusto-KC7/activity-0.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Activity 0: Login & Setup of the Cyber Environment"
+description: ""
```

Comment:
Feel free to remove the description for all the activities (i.e. non index page)

```suggestion
```

Reply:
ok done